internal/querynode/task_search.go

```diff
@@ -181,7 +181,9 @@ func (s *searchTask) estimateCPUUsage() {
 		segmentNum := int64(len(s.req.GetSegmentIDs()))
 		s.cpu = int32(s.NQ * segmentNum / 2)
 	}
-	if s.cpu > s.maxCPU {
+	if s.cpu <= 0 {
+		s.cpu = 5
```

Comment:
NIT: Magic number. Shall we put it into configure file or a const?

Reply:
These parameters are adjusted after testing and can be configurable in next pr.